complete/src/main/java/com/example/restservice/service/TokenService.java

```diff
@@ -75,6 +76,9 @@ public TokenDetailResponse createToken(CreateTokenRequest createTokenRequest) {
             .findById(createTokenRequest.getQueueId())
             .map(
                 queue -> {
+                  if(queue.getStatus().equals(QueueStatus.PAUSED)) {
```

Comment:
let's also fail if the queue has been deleted

Reply:
open to discussion on this one, though. If a queue was paused once, should we allow it to be paused again.? I think not. Coz if it's called again, there is a bug in the UI?